undefined_syms.txt

```diff
@@ -172,6 +172,7 @@ D_80151328 = 0x80151328;
 gCurrentDoorSoundsSet = 0x80151340;
 
 gTriggerCount = 0x80151334;
+D_80156954 = 0x80156954;
```

Comment:
Please also add an entry to symbol_addrs.txt. I'll be making the formats more similar soon..

Reply:
What if we wrote a tool to convert between the two? I've definitely forgotten to add to symbol_addrs.txt multiple times since we added it

Reply:
Only add `D_80156954;0x80156954` there or add `func_8013A854;0x8013A854` too?